### PR TITLE
fix: Remove Netlify CLI from dev dependencies

### PR DESCRIPTION
**Pull request by coder12git**
## Description
This PR  fixes issue #2170

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [x] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by nickytonline** [comment]
Thanks for the PR @coder12git! Aside from removing the dependency, we'll also need to [update the installation instructions](https://github.com/open-sauced/app/issues/2170#issuecomment-1816766017) to mention installing the Netlify CLI globally.

**Comment by coder12git** [comment]
@nickytonline i am not sure, where should i add docs about installing the Netlify CLI globally, also it's my first contribution here, it will be great if you can let me know which will be the right place. Also my suggestion will be to add it in the Readme under [this](https://github.com/open-sauced/app#%EF%B8%8F-local-development) section as a note.

**Comment by coder12git** [comment]
> Thanks for the PR @coder12git! Aside from removing the dependency, we'll also need to [update the installation instructions](https://github.com/open-sauced/app/issues/2170#issuecomment-1816766017) to mention installing the Netlify CLI globally.

Can you please tell me where should i add this? I am thinking to add this [here](https://github.com/open-sauced/app#%EF%B8%8F-local-development)

**Comment by nickytonline** [comment]
> > Thanks for the PR @coder12git! Aside from removing the dependency, we'll also need to [update the installation instructions](https://github.com/open-sauced/app/issues/2170#issuecomment-1816766017) to mention installing the Netlify CLI globally.
> 
> Can you please tell me where should i add this? I am thinking to add this [here](https://github.com/open-sauced/app#%EF%B8%8F-local-development)

That looks like a good spot to me. cc: @BekahHW

One other thing we can do since we're cleaning things up in the docs, is replace `npm run dev:netlify` with `netlify dev` and we can also remove this command from the package.json.

```diff
  "scripts": {
    "types:auto-add": "npx typesync",
    "types:auto-check": "npm run types:auto-add -- --dry",
    "preinstall": "npx @open-sauced/check-engines",
    "prestart": "npx @open-sauced/check-engines",
    "push": "npx @open-sauced/conventional-commit",
    "dev": "next dev",
-   "dev:netlify": "netlify dev",
...
```

**Comment by coder12git** [comment]
@nickytonline done the changes. Please review

**Review by OgDev-01 — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -49,10 +49,12 @@ npm run dev
 
 ### ☁️️ Netlify Edge Functions
 
+**Note**: For seamless integration and optimal local development experience, it is recommended to have the `Netlify CLI` installed globally. Follow the [installation guide]( https://docs.netlify.com/cli/get-started/) to set up Netlify CLI on your machine.
```
Comment (nickytonline) [inline README.md:52]: @BekahHW just checking in here for your thoughts on wording,

**Comment by coder12git** [comment]
@nickytonline i am looking to work on frontend issues here, but seems like most of issues are made for core-team-members here, so please can you let me know, which will be the right place for me to find those issues.

**Comment by nickytonline** [comment]
> @nickytonline i am looking to work on frontend issues here, but seems like most of issues are made for core-team-members here, so please can you let me know, which will be the right place for me to find those issues.

This is what's available at the moment, https://github.com/open-sauced/app/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+no%3Aassignee+label%3A%22%F0%9F%90%9B+bug%22+-label%3A%22%F0%9F%91%80+needs+triage%22+-label%3A%22core+team+work%22+ .

**Comment by BekahHW** [comment]
Let's simplify this part "For seamless integration and optimal local development experience" with 

"For the best local development experience,"

**Comment by coder12git** [comment]
> Let's simplify this part "For seamless integration and optimal local development experience" with
> 
> "For the best local development experience,"

Done the changes.

**Review by nickytonline — APPROVED** [review]
Thanks for the pull request @coder12git! :shipit:

**Pull request merged**